.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -16,6 +16,22 @@ parameters:
   default: true
 
 steps:
+- bash: |
+        echo "Removing existing go environment"
+        sudo rm -r /usr/local/go
+        GOLANG_VERSION="go1.18.5"
+        echo "Downloading ${GOLANG_VERSION}"
+        curl -O "https://dl.google.com/go/${GOLANG_VERSION}.linux-amd64.tar.gz"
+        
+        echo "removing existing go environment"
+        sudo rm -r /usr/local/go
```

Comment:
already on L21?

Reply:
@cameronmeissner removed L26 and 27 instead 😆